metadata-ingestion/src/datahub/ingestion/source/unity/config.py

```diff
@@ -50,9 +50,9 @@ class UnityCatalogSourceConfig(StatefulIngestionConfigBase, DatasetSourceConfigM
         description="Option to enable/disable lineage generation.",
     )
 
-    include_table_ownership: bool = pydantic.Field(
+    include_ownership: bool = pydantic.Field(
```

Comment:
let's add a `pydantic_field_renamed` helper to preserve backwards compatibility

Reply:
+1 on that. Will merge after that